UnityProject/Assets/Scripts/UI/Systems/Lobby/CharacterCustomization.cs

```diff
@@ -130,7 +131,7 @@ private void GrabAllSpecies()
 				Loggy.LogError("UNABLE TO GRAB ALL SPECIES!! CHARACTER CREATION SCREEN IS SURELY GOING TO BE BROKEN!!!");
 				return;
 			}
-			allSpecies = RaceSOSingleton.Instance.Races.Where(alien => alien.Base.CanBePlayerChosen).ToList();
+			allSpecies = RaceSOSingleton.GetPlayerRaces();
```

Comment:
wait, you did change this perfectly fine LINQ with that new method, why?

Reply:
We needed a general function to grab all player species without rewriting the same code over and over again.

Reply:
fair, but I still consider the LINQ version to be nicer.